src/renderer/ScreenController.tsx

```diff
@@ -114,6 +129,24 @@ export default class ScreenController extends Component {
             <p>{this.state.message.text}</p>
           </div>
         )}
+        {!this.state.online && (
+          <div className='no-network-toast'>
+            <b>Offline</b>
+            <div
+              onClick={() => {
+                this.setState({ tryConnectCooldown: false })
+                setTimeout(
+                  () => this.setState({ tryConnectCooldown: true }),
+                  15000
+                )
+                setTimeout(() => DeltaBackend.call('context.maybeNetwork'), 100)
+              }}
+              className={this.state.tryConnectCooldown ? '' : 'disabled'}
+            >
+              Try to connect now
+            </div>
+          </div>
+        )}
```

Comment:
Also I would like to have this code in a seperate component.
`<OfflineToast />`

This way the component could keep all the state about being online or offline and just listen to the DC_NETWORK_ERROR and if it's offline, also to the SMTP_CONNECTED... things. Otherwise the ScreenController will grow to a beast the same way Settings did.